riva/proto/riva_asr.proto

```diff
@@ -195,6 +206,10 @@ message RecognitionConfig {
   // configuration options to plugins used in the
   // model pipeline.
   map<string, string> custom_configuration = 24;
+
+  // Option to configure End of Utterance parameters.
+  // If empty, Riva will use default values.
```

Comment:
Update comment to:

// Config for tuning start or end of utterance parameters.
// If empty, Riva will use default values or custom values if specified in `riva-build` arguments.

Reply:
Updated please check